### ISSUE TITLE
pod2man install issue on 10.6

### ISSUE THREAD

**Issue by skinkie** [Homebrew/homebrew#47464]
As follow up of #46566 

https://gist.github.com/anonymous/52ea640338d2ba87dd6d

```
==> Installing dependencies for wget: pod2man
==> Installing wget dependency: pod2man
==> Downloading http://archives.eyrie.org/software/perl/podlators-4.03.tar.xz
Already downloaded: /Library/Caches/Homebrew/pod2man-4.03.tar.xz
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/pod2man/4.03
==> make install
Error: No such file or directory - /usr/local/Cellar/pod2man/4.03/man
```

**Comment by mistydemeo** [comment]
Looks like the manpages didn't get installed, so trying to move them is failing.

**Comment by mistydemeo** [comment]
Should be fixed by bc09f92e42426acfd80f8d2619444d05fcac209c.
